frontend/relay.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require("path");
+const { APP_PATH, OUT_PATH } = require("./constants");
+
+module.exports = {
+    src: APP_PATH,
+    schema: path.join(OUT_PATH, "schema.graphql"),
+    language: "typescript",
+    artifactDirectory: path.join(APP_PATH, "queries"),
```

Comment:
The gitignore and `frontend/src/query-types/README.md` are using `query-types` as directory. This is a mistake, right?